torch/fx/passes/runtime_assert.py

```diff
@@ -258,14 +258,24 @@ def add_runtime_asserts(ras):
                 # nodes
                 with _set_node_metadata_hook(gm, _node_metadata_hook):
                     res = _sympy_interp(expr_to_proxy, ra.expr).node
+                    expr_msg = f"Runtime assertion failed for expression {ra.expr} on node '{res}'"
+                    ra_msg = None
+                    if ra.msg:
+                        if callable(ra.msg):
+                            try:
+                                ra_msg = ra.msg()
+                            except:
+                                pass
+                        else:
+                            assert isinstance(ra.msg, str)
+                            ra_msg = ra.msg
+                    if ra_msg is not None:
+                        assert_msg = f"{expr_msg}\n\nThe original traceback points to the following location and error message:\n{ra.loc}\n{ra_msg}"
```

Comment:
maybe with out new lines? this is how it show up when the graph is printed 
![Uploading Screenshot 2025-04-22 at 8.36.47 AM.png…]()